diff --git a/llm/llama.cpp/generate_darwin_amd64.go b/llm/llama.cpp/generate_darwin_amd64.go
index 2106507..9c72455 100644
--- a/llm/llama.cpp/generate_darwin_amd64.go
+++ b/llm/llama.cpp/generate_darwin_amd64.go
@@ -6,13 +6,13 @@ package llm
 //go:generate git -C ggml apply ../patches/0003-metal-fix-synchronization-in-new-matrix-multiplicati.patch
 //go:generate git -C ggml apply ../patches/0004-metal-add-missing-barriers-for-mul-mat-2699.patch
 //go:generate git -C ggml apply ${RECIPE_DIR}/ggml-metal-pick-device.patch
-//go:generate cmake -S ggml -B ggml/build/cpu -DLLAMA_METAL=off -DLLAMA_ACCELERATE=on -DLLAMA_K_QUANTS=on -DCMAKE_EXE_LINKER_FLAGS="-static-libstdc++"
+//go:generate cmake -S ggml -B ggml/build/cpu -DLLAMA_METAL=off -DLLAMA_ACCELERATE=OFF -DLLAMA_K_QUANTS=on -DCMAKE_EXE_LINKER_FLAGS="-static-libstdc++" -DLLAMA_BLAS=ON -DLLAMA_BLAS_VENDOR=Intel10_64_dynk
 //go:generate cmake --build ggml/build/cpu --target server --config Release
 //go:generate mv ggml/build/cpu/bin/server ggml/build/cpu/bin/ollama-runner
 
 //go:generate git -C gguf status
 //go:generate git -C gguf apply ../patches/0001-update-default-log-target.patch
 //go:generate git -C gguf apply ${RECIPE_DIR}/gguf-metal-pick-device.patch
-//go:generate cmake -S gguf -B gguf/build/cpu -DLLAMA_METAL=off -DLLAMA_ACCELERATE=on -DLLAMA_K_QUANTS=on -DLLAMA_NATIVE=off -DLLAMA_AVX=on -DLLAMA_AVX2=off -DLLAMA_AVX512=off -DLLAMA_FMA=off -DLLAMA_F16C=on -DCMAKE_EXE_LINKER_FLAGS="-static-libstdc++"
+//go:generate cmake -S gguf -B gguf/build/cpu -DLLAMA_METAL=off -DLLAMA_ACCELERATE=OFF -DLLAMA_K_QUANTS=on -DLLAMA_NATIVE=off -DLLAMA_AVX=on -DLLAMA_AVX2=off -DLLAMA_AVX512=off -DLLAMA_FMA=off -DLLAMA_F16C=on -DCMAKE_EXE_LINKER_FLAGS="-static-libstdc++" -DLLAMA_BLAS=ON -DLLAMA_BLAS_VENDOR=Intel10_64_dynk
 //go:generate cmake --build gguf/build/cpu --target server --config Release
 //go:generate mv gguf/build/cpu/bin/server gguf/build/cpu/bin/ollama-runner
