diff --git a/ggml-metal.m b/ggml-metal.m
index d23fff1..56264a3 100644
--- a/ggml-metal.m
+++ b/ggml-metal.m
@@ -109,10 +109,29 @@ @implementation GGMLMetalClass
 struct ggml_metal_context * ggml_metal_init(int n_cb) {
     fprintf(stderr, "%s: allocating\n", __func__);
 
+    id <MTLDevice> device;
+    id <MTLDevice> dev;
+    NSArray * devices = MTLCopyAllDevices();
+
+    device = MTLCreateSystemDefaultDevice();
+    for (dev in devices) {
+        ;
+        if ([dev isRemovable]) {
+            ;
+        } else if ([dev isLowPower]) {
+            ;
+        } else {
+            ;
+            device = dev;
+        }
+    }
+    
+    
+
     struct ggml_metal_context * ctx = malloc(sizeof(struct ggml_metal_context));
 
     ctx->n_cb   = n_cb;
-    ctx->device = MTLCreateSystemDefaultDevice();
+    ctx->device = device;
     ctx->queue  = [ctx->device newCommandQueue];
     ctx->n_buffers = 0;
     ctx->concur_list_len = 0;
